client/components/Scenario/index.jsx

```diff
@@ -34,7 +34,10 @@ class Scenario extends Component {
     }
 
     isScenarioRun() {
```

Comment:
I had a thought for refectoring this at one point that might be worth doing now. What if we pass `isScenarioRun` as a boolean prop to `Scenario` based on our app logic instead of determining it from (potentially increasingly complex) URLs? So like here we could pass it to the Scenario component and likewise in routes for the `"/moment/:scenarioId"` route. [This could be one way to go about it in the router.](https://tylermcginnis.com/react-router-pass-props-to-components/)